src/modules/RA/BaseLayer/components/ZoomInput.js

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback, useMemo } from 'react';
+import { useField, useForm } from 'react-final-form';
+
+import RangeInput from '../../../../components/react-admin/RangeInput';
+
+const ZoomInput = ({
+  basePath,
+  record,
+  resource,
+  initialValue: [valueMin, valueMax],
+  fieldPaths: [fieldMin = 'min_zoom', fieldMax = 'max_zoom'] = [],
+  ...props
+}) => {
+  const form = useForm();
+
+  const {
+    input: {
+      value: min,
+    },
+  } = useField(fieldMin);
+  const {
+    input: {
+      value: max,
+    },
+  } = useField(fieldMax);
+
+  const minZoom = useMemo(() => (
+    Number.isFinite(min) ? min : valueMin
+  ), [min, valueMin]);
+
+  const maxZoom = useMemo(() => (
+    Number.isFinite(max) ? max : valueMax
+  ), [max, valueMax]);
+
+  const onZoomChange = useCallback((_, [newMin, newMax]) => {
+    form.change('min_zoom', newMin);
+    form.change('max_zoom', newMax);
+  }, [form]);
+
+  return (
+    <RangeInput
+      {...props}
+      onChange={onZoomChange}
+      value={[minZoom, maxZoom]}
```

Comment:
This value should be memoized as the list is created at each render.

Reply:
I think you can replace the two use memo with one for the list...

Reply:
Ok done